src/patternfly/components/Page/page-header-brand-link.hbs

```diff
@@ -1,6 +1,6 @@
-<a href="{{page-header-brand-link--href}}" class="pf-c-page__header-brand-link{{#if page-header-brand-link--modifier}} {{page-header-brand-link--modifier}}{{/if}}"
+<{{#if page-header-brand-link--href}}a href="{{#if page-header-brand-link--href}}{{page-header-brand-link--href}}{{else}}#{{/if}}" {{else}}span {{/if}} class="pf-c-page__header-brand-link{{#if page-header-brand-link--modifier}} {{page-header-brand-link--modifier}}{{/if}}"
```

Comment:
won't this inner test will always evaluate true since it's the same test as the outer test?

Reply:
good point @mcoker - fixed